src/sparsezoo/v2/test.md

```diff
@@ -0,0 +1,100 @@
+# API User pathway
+
+## Download one file
+
+### Through the string arguments
+```python
+from sparsezoo import Model
+
+stub = "zoo:cv/classification/mobilenet_v1-1.0/pytorch/sparseml/imagenet/pruned-moderate"
+stub = stub + "?" + "recipe=transfer_learn"
+model = Model(path=stub)
+model.download() # will only download only recipe file (as indicated by the string argument). If no argument provided, .download() will download to cache
+recipes: str = model.recipes.downloaded_path() 
+...
+```
+```python
+from sparsezoo import Model
+
+stub = "zoo:cv/classification/mobilenet_v1-1.0/pytorch/sparseml/imagenet/pruned-moderate"
+stub = stub + "?" + "deployment" # if we want the whole directory, the parameter does not have to have value
+model = Model(path=stub)
+model.download()
+deployment: str = model.recipes.downloaded_path() 
```

Comment:
similarly to the v1 api let's make `downloaded_path` call `download()` if the file is not downloaded yet, that way we can save the extra call to `downlaod()` on line 22

Reply:
good idea